molgenis-biobank-plots/src/main/frontend/biobank-plots/src/components/Graph.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <vue-chart :columns="columns" :rows="aggs" :options="options"
+             chartType="BarChart" :chartEvents="chartEvents"
+             ref="sampleCounts"></vue-chart>
+</template>
+
+<script>
+  import {mapState} from 'vuex'
+  import Vue from 'vue'
+  import VueCharts from 'vue-charts'
+
+  Vue.use(VueCharts)
+
+  export default {
+    data: function () {
+      const self = this
+      return {
+        chartEvents: {
```

Comment:
Ehh.. a function in data, if you place it in methods() you can call it from the template in the same way